src/cards/LibraryFile.ts

```diff
@@ -12,7 +12,7 @@ import { ManualPromise } from '../utils/misc/ManualPromise'
 import { getPngMetadataFromUint8Array } from '../utils/png/_getPngMetadata'
 import { Library } from './Library'
 
-import { dirname } from 'path'
+import { dirname } from 'pathe'

```

Comment:
pathe? - oh that is a nice library, I'll have to start using that

Reply:
just replaced all remaining path a few minutes ago; I'll try to add some lint rule that prevent using it; or maybe I'll just alias path to pathe directly instead :)